#requset
hyundai_steel_crane_automation_system/HeadCR head
hyundai_steel_crane_automation_system/ACWL0001Body body
---
#response
bool accepted
